plenum/test/consensus/view_change/test_sim_view_change.py

```diff
@@ -78,10 +85,13 @@ def test_new_view_combinations(random):
     # Check that all committed requests are present in final batches
     for _ in range(10):
         num_votes = quorums.strong.value
-        votes = random.sample(view_change_messages, num_votes)
+        votes = custom_random.sample(view_change_messages, num_votes)
 
         cp = pool.nodes[0]._view_changer._new_view_builder.calc_checkpoint(votes)
-        assert cp is not None
+        # TODO: unskip after fixing the issue #1506
```

Comment:
Issue https://github.com/hyperledger/indy-node/pull/1506 was fixed.

Reply:
This is a PR, not an issue.
Issue:  https://github.com/hyperledger/indy-plenum/issues/1506 (view change bug)